### DISCUSSION TITLE
Do we have self-hosted macOS runners?

### DISCUSSION THREAD

**Discussion by yurikoles** [Homebrew/discussions#2538] [Writing Formulae/Casks] [answered]
Runtime test of `svt-hevc` in this PR https://github.com/Homebrew/homebrew-core/pull/52271 failed because of too old CPU on macOS runner. It requires Broadwell, but we have Penryn.

Are there any changes in infrastructure that can resolve this issue?

**Comment by SMillerDev** [comment]
We have self hosted runners but if it's too old on CI (where we have relatively new hardware) that means it'll also be too old for most users.

**Comment by Bo98** [comment, marked as answer]
We run in a VM so it's probably a consequence of that where it appears as Penryn. I think the underlying hardware is Ivy Bridge (2013 Mac Pro) but that's just from memory and I'm assuming that hasn't changed.

But with that said, we support Nehalem and newer and do not currently support enforcing CPU-based requirements beyond `depends_on arch: ...`.
